testing/types.go

```diff
@@ -51,6 +51,7 @@ type JwtBody struct {
 	Subject string   `json:"sub" yaml:"sub"`
 	Roles   []string `json:"roles" yaml:"roles"`
 	Issuer  string   `json:"iss" yaml:"iss"`
+	Claims map[string]interface{} `json:"claims,omitempty" yaml:"claims,omitempty"`
```

Comment:
please make sure your IDE runs `go fmt` upon saving (or please run it before merging)